scanamabob/commands/scan.py

```diff
@@ -11,7 +11,7 @@
 import scanamabob.scans.rds as rds
 import scanamabob.scans.redshift as redshift
 from scanamabob.context import Context, add_context_to_argparse
-
+import IPython
```

Comment:
I think you forgot to remove this

Reply:
I think it got removed in 4ee3f457ca37696ce1cc1d6e225a07355c1c703d.

Reply:
and added again in https://github.com/CarveSystems/scanamabob/pull/35/commits/fc954e33f17ae6c2a0681ebc484a964667eef399